### ISSUE TITLE
Does not remove all components 

### ISSUE THREAD

**Issue by modulexcite** [Microsoft/VisualStudioUninstaller#48]
The following components are not  removed by the uninstaller.
- Xamarin 
- Android SDK
- Java SDK
- .NET Core Preview
- .NET Multi-Targeting Packs and SDKs
- SSDT Tools

It leaves behind a lot of packages that are part of Visual Studio 2015. I'm not sure if these should have been removed by this.
Perhaps you should add a note here on what we should expect to be removed.

**Comment by tobyhu87** [comment]
Hi,

That is good suggestions.  The uninstaller is not designed to removed SDKs and non-Microsoft products.  Because we do not know if you are using those items outside of Visual Studio.

thanks
toby.

**Comment by modulexcite** [comment]
Isn't Xamarin now part of VS 2015 and what about the SSDT Tools? I'm not sure how I would be using them without VS. Perhaps a command line option to force remove everything or other products as necessary might be a good option.

**Comment by bricelam** [comment]
+1 my expectations were not met. This led to disappointment and, as usual, reimaging.

**Comment by modulexcite** [comment]
I agree, I think they should put that information here on what components we should expect to be removed.

**Comment by qaqz111** [comment]
Agreed.

> The uninstaller is not designed to removed SDKs and non-Microsoft products. Because we do not know if you are using those items outside of Visual Studio.

That's why a **command line option** is needed but not a MUST feature.

**Comment by modulexcite** [comment]
One can always provide that as an option to remove(or not) other components such as sdks.
